pkg/apis/istio/v1alpha1/session_types.go

```diff
@@ -8,12 +8,27 @@ import (
 
```

Comment:
Maybe we can have a make target doing that for us? (and make it as default make target)

In other words: "shall we make make making those changes on our behalf as part of default make's make" ;)